roles/filetree_create/templates/current_inventories.j2

```diff
@@ -1,35 +1,65 @@
-{% if (first_inventory | default(true) | bool) and (current_inventory_index | default(0) == 0) %}
+{% if (first_inventory | default(true) | bool) or (current_inventory_index | default(0) == 0) %}
```

Comment:
Did you tested that change? It is not related with the PR itself, so this is why i'm asking...

Reply:
Change done by mistake, reverted